recipes/faac/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
Please, use `cmake_find_package` to consume the library
```suggestion
    generators = "cmake", "cmake_find_package"
```